package.json

```diff
@@ -19,6 +19,7 @@
     "config": "^3.3.6",
     "cookie-parser": "~1.4.5",
     "cors": "^2.8.5",
+    "crypto": "^1.0.1",
```

Comment:
I thought crypto was built in to nodejs, do we need to install it as a package?

Look into the last line of overview: https://www.scaler.com/topics/nodejs/cryptography-in-nodejs/
or
go here: https://developer.mozilla.org/en-US/docs/Web/API/Crypto and look into the first line after the **Note**

Reply:
Yaa, I was facing some issues before adding it but it was some else issue. Pushed it mistakenly. Thank you. Fixed it.